cdap-explore-client/src/main/java/co/cask/cdap/explore/service/Explore.java

```diff
@@ -116,6 +116,14 @@
    */
   List<QueryInfo> getQueries(Id.Namespace namespace) throws ExploreException, SQLException;
 
+  /**
+   * Get the number of active queries being executed in Hive.
```

Comment:
Active queries are queries that are being executed, or completed queries that have not been closed yet.